pkg/constants/constants.go

```diff
@@ -9,8 +9,8 @@ const (
 
 	BaseDirName = ".avalanche-cli"
 
-	SubnetEVMReleaseVersion   = "v0.2.3"
-	AvalancheGoReleaseVersion = "v1.7.13"
+	SubnetEVMReleaseVersion   = "v0.2.5"
+	AvalancheGoReleaseVersion = "v1.7.14"
```

Comment:
Should we actually bump this to v1.7.16 already?

Reply:
fixed